src/gui/dialogs/VersionedSaveDialog.cpp

```diff
@@ -63,6 +65,17 @@ VersionedSaveDialog::VersionedSaveDialog( QWidget *parent,
 	hLayout->addWidget( minusButton );
 	layout->addLayout( hLayout, 2, 1 );
 
+	if (saveOptionsWidget) {
+		auto *groupBox = new QGroupBox(tr("Save Options"));
+		auto optionsLayout = new QGridLayout;
+
+		optionsLayout->addWidget(saveOptionsWidget, 0, 0, Qt::AlignLeft);
+
+		groupBox->setLayout(optionsLayout);
+
+		layout->addWidget(groupBox, layout->rowCount()+1, 0, 1, -1);
```

Comment:
```suggestion
		layout->addWidget(groupBox, layout->rowCount() + 1, 0, 1, -1);
```
From our coding convention:
```
• Infix operators ('=', '+', '-', '*', '/', etc.) should have a space before and after them.
```